MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,7 +2,153 @@ import UIKit
+    QuizeQuestion(image: "Tesla", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+    QuizeQuestion(image: "Vivarium", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false)
+    ]
+    
+    @IBOutlet weak var movieImageView: UIImageView!
+    @IBOutlet weak var questionLabel: UILabel!
+    @IBOutlet private weak var counterLabel: UILabel!
+    @IBOutlet private weak var noButton: UIButton!
+    @IBOutlet private weak var yesButton: UIButton!
+    
+    private func showStep(quize step: QuizeStepViewModel) {
+        noButton.isUserInteractionEnabled = true
+        yesButton.isUserInteractionEnabled = true
+        movieImageView.image = step.image
+        questionLabel.text = step.question
+        counterLabel.text = step.questionNumber
+    }
+    
+    private func showResult(quize result: QuizeResultsViewModel) {
+        noButton.isUserInteractionEnabled = false
+        yesButton.isUserInteractionEnabled = false
+        let alert = UIAlertController(
+            title: result.title,
+            message: result.text,
+            preferredStyle: .alert
+        )
+        let action = UIAlertAction(
+            title: result.buttonText,
+            style: .default, handler: {_ in
+            self.currentQuestionIndex = 0
+            self.rightAnswerCount = 0
+            self.showStep(quize: self.createStepModel(model: self.questions[0]))
+        })
+        alert.addAction(action)
+
+        present(alert, animated: true, completion: nil)
+    }
+    
+    private func createStepModel(model: QuizeQuestion) -> QuizeStepViewModel {
+        return QuizeStepViewModel(
+            image: UIImage(named: model.image) ?? UIImage(),
+            question: model.text,
+            questionNumber: "\(currentQuestionIndex + 1)/\(questions.count)"
+        )
+    }
+    
+    private func showAnswerResult(isCorrect: Bool) {
+        noButton.isUserInteractionEnabled = false
+        yesButton.isUserInteractionEnabled = false
+        movieImageView.layer.borderWidth = 8
+        movieImageView.layer.borderColor = isCorrect ? UIColor.ypGreen.cgColor : UIColor.ypRed.cgColor
+        if isCorrect {
+            rightAnswerCount += 1
+        }
+    }
+    private func getResultMessage() -> String {
+        let formater = DateFormatter()
+        formater.dateFormat = "dd.MM.yyyy hh:mm"
+        let result: String = "Ваш результат: \(rightAnswerCount)/\(questions.count)."
+        let quize: String = "Количество сыгранных квизов: \(quizeCount)."
+        let record: String = "Рекорд: \(answersRecord)/\(questions.count) (\(formater.string(from: recordDate)))."
+        let percent = (Double(totalRightAnswerCount) / Double(quizeCount * questions.count) * 10000).rounded() / 100
+        let statistic: String = "Средняя точность: \(percent)%."
+        return result + "\n" + quize + "\n" + record + "\n" + statistic
+    }
+    private func showNextQuestionOrResults() {
```

Comment:
```suggestion
    }
    
    private func showNextQuestionOrResults() {
```

Не забывайте оставлять немного пространства между блоками кода, так его легче воспринимать